src/configuration.h

```diff
@@ -139,7 +139,9 @@ class MetadataAgentConfiguration {
   }
 
  private:
+  friend class MetadataAgentConfigurationTest;
```

Comment:
Please add a blank line after this to visually separate it from the functions.

Reply:
Done.